basicsr/archs/ddcolor_arch.py

```diff
@@ -383,3 +385,34 @@ def forward(self, img_features, encode_feat):
         color_preds = torch.einsum('bqc,bchw->bqhw', color_embed, img_features)
         return color_preds
 
+
+class DDColorHF(DDColor, PyTorchModelHubMixin):
+    def __init__(self, config):
+        super().__init__(**config)
+    
+    def forward(self, x):
+        super().forward(x)
+
+
+config = dict(
+    encoder_name='convnext-t',
+    decoder_name='MultiScaleColorDecoder',
+    input_size=[512, 512],
+    num_output_channels=2,
+    last_norm='Spectral',
+    do_normalize=False,
+    num_queries=100,
+    num_scales=3,
+    dec_layers=9,
+)
```

Comment:
This example could be added to the README.md, perhaps by adding a new section "Inference with HuggingFace" under "Quick Start". In any case, I don't think it should be included within the package itself.